CRM/Core/ShowHideBlocks.php

```diff
@@ -82,8 +82,8 @@ public function addToTemplate() {
 
     $template = CRM_Core_Smarty::singleton();
     $template->ensureVariablesAreAssigned(['elemType']);
-    $template->assign_by_ref('hideBlocks', $hide);
-    $template->assign_by_ref('showBlocks', $show);
+    $template->assign('hideBlocks', $hide);
```

Comment:
unrelated in the end as I removed the tpl include but this does need to happen for Smarty5